src/Simulation/EntryPointDriver/Driver.cs

```diff
@@ -208,6 +214,41 @@ private void AddOptionIfAvailable<T>(Command command, OptionInfo<T> option)
                     $"The required option {option.Aliases.First()} conflicts with an entry point parameter name.");
             }
         }
+
+        /// <summary>
+        /// Adds a validator to the command such that the specified options cannot be simultaneously included.
+        /// </summary>
+        /// <param name="command">The command to add the validator to.</param>
+        /// <param name="primaryAliases">The primary aliases of the options to be marked as mutually exclusive.</param>
+        /// <remarks>Options must already be added to the command before invoking this method.</remarks>
```

Comment:
I'm not sure that this remarks section is true - as long as the validator is added before the command line is parsed, I don't think the order relative to the options being added matters?

Reply:
You are right! Thanks for catching this, I will remove the remarks section.

Reply:
Updated.